packages/kbn-securitysolution-autocomplete/src/field/__tests__/index.test.tsx

```diff
@@ -7,17 +7,18 @@
  */
```

Comment:
thanks @rylnd for looking into it :) 
Please if you have convention define anywhere share it with me.

This convention I tried to use to 
1. Organise the component folder a bit, and allow easy access to the desired folder 
2. Usually the Jest Code coverage collects and ignores this `__tests__` folder and while I was looking into one of the component's test-coverage result  I found
 https://kibana-coverage.elastic.dev/2022-08-31T05:52:00Z/jest-combined/x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_item_card/index.html
 
![image](https://user-images.githubusercontent.com/12671903/187723795-29fadd70-cc92-4ecc-8dab-aed91a47ef0a.png)

Which seems that we collect  test-coverage from .test as well? please correct me if I am wrong

Reply:
Oh wow, great find on the coverage report; that sure looks like a bug to me. I think you might've found a great way to bump our code coverage numbers 😉 

I don't have a strong preference for whether there is/isn't a `__tests__` folder, I just wanted to point out that it wasn't necessary 😄.

Reply:
😄 yeah it looks like it doesn't collect the coverage correctly.

Great let's see if it helps 🤞

Reply:
@WafaaNasr digging deeper this may be a bug in our jest config; we have a lot of files [this one](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/jest.config.js#L15), which do not specify patterns to negate as is typical in such a config. If the base `collectCoverageFrom` config is overwritten here, that might be the issue.

Reply:
Thanks @rylnd for digging into this bug, great you find the root cause it seems I got confuse because of a `jest=-preset.js` file under `bazel-bin/packages/kbn-test/jest-preset.js` 
it has 
```
 // An array of regexp pattern strings that are matched against all source file paths, matched files to include/exclude for code coverage
  collectCoverageFrom: [
    '**/*.{js,mjs,jsx,ts,tsx}',
    '!**/{__test__,__snapshots__,__examples__,*mock*,tests,test_helpers,integration_tests,types}/**/*',
    '!**/*mock*.{ts,tsx}',
    '!**/*.test.{ts,tsx}',
    '!**/*.d.ts',
    '!**/index.{js,ts,tsx}',
  ],
 ```
 I already shared this with the team in our sync and we started this [thread](https://elastic.slack.com/archives/C0TR0FAET/p1661967036736909) to know the best-practice, I am fine with any agreement here ^_^ just looking for an alignment

I would be fine to fix this config here in this PR :) as well and remove the `__tests__` folder